src/bidsGZipRawFolder.m

```diff
@@ -0,0 +1,41 @@
+% (C) Copyright 2020 CPP BIDS SPM-pipeline developers
+
+function bidsGZipRawFolder(opt, keepUnzippedNii)
+  %
+  %
+  % GZip the nii files in a ``raw`` bids folders from the. It will do it independently of the task.
```

Comment:
from the  ??? 

the suspense is unbearable

:wink:

Reply:
I don't remember what I meant with that 😅
```suggestion
  % GZip the nii files in a ``raw`` bids folders. It will do it independently of the task.
```